### ISSUE TITLE
I need some help about the setup (disk, etc.)

### ISSUE THREAD

**Issue by giobbino** [jbilander/sdbox#19]
Hi all,
I've a couple of Amiga 500 _(one donated, one I got from a litter box -literally!- and fixed)_ but I have no skill/knowledge about Amiga _(I'm most on 8-bit Commodore computers)_. One has the Kickstart 1.2 so I have to update it; the other has the KS 1.3 + WB 1.3.3 so this one should be compatible with SDbox.

I'm updating my post since I finally make the Lotharek drive working as DF0, but I can't make a working image with the needed files (it boots from the sdcard but then I can't copy it  on a floppy using the physical drive DF1, it says the image I've built is someway corrupted. 

Long story short **: does anybody here has an image (.ADF or .HFE) of a bootable KS 1.3 with the needed files on it?**
I'm not planning to put the lotharek (or gotek) virtual drive inside the A500, since the FDD is working fine and I've tons of floppies here.

**Comment by jbilander** [comment]
Hi, yes I have an ADF-image of WB 1.3 with everything configured correctly. I would love to share it but WB (and KS) is not free software unfortunately. Is your SDBox a version 1 or version 2? The device driver (_spisd.device_) is different between the two.

**Comment by giobbino** [comment]
Hi,
thanks for your answer!
yes I understand; WB is freely available on the web though (and honestly it's abandoned software...). However I understand if you are not comfortable to share the disk.
I've built the V.1 because I have everything at home (SD card reader and Arduino Nano). 
Since eventually I got my Lotharek FDD emulator working, I'm trying to make an image with the needed files, but it's giving me some error after the boot: I'm getting some "disk failure" and I don't know why. Maybe the .hfe images made by the HxCFloppyEmulator (dropping files inside) aren't ok?

**Comment by jbilander** [comment]
Hi, you would think it's abandonware but no Cloanto owns the rights and still sells both WB and Kickstarts in the Amiga Forever pack. Also Hyperion sells new WB/KS 3.2. Drop me an email and it's easier for me to help out: jorgen.bilander at gmail com

**Comment by giobbino** [comment]
Jorgen, thank you for your help. After I mounted sd0: and tried a DIR command, I get the "Could not get information for SD0: Error 115" as described in the https://github.com/jbilander/sdbox/issues/2 thread. 
I'm trying with different SD cards / FAT-FAT32 / partition sizes etc.

I have no clue from the LEDs... since I have no LED (yet) on my SDbox. I'm waiting for the transistor from Aliexpress (I didn't find them neither at the local store, nor on-line for a reasonable price here in Brazil). Actually my board has just the Nano and the SD card with the pullup resistor as described by Sukkopera. 

At least, thank to you, I can focus on the board! :-)    
Ah, I think I will also try to replace the 8520, just in case the installed one is bad (luckily I have a couple spare, here)

Will let you know later!

**Comment by giobbino** [comment]
Success! I'm opening a new thread (some kind of a tutorial) talking about my problems and doubts and how I solved.
